books/Dsa in c++ by narsimha

```diff
@@ -0,0 +1,2 @@
+https://github.com/Humblefool76/c-/blob/daf11b6e2a885a0e8e612f0f4b159acf443afd82/Data%20Structures%20and%20Algorithms%20-%20Narasimha%20Karumanchi_compressed.pdf
```

Comment:
Please provide official links only. Add the links to the proper file

Read [Contributing.md](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md) for more details